code/game/turfs/turf.dm

```diff
@@ -111,6 +112,9 @@
 		var/atom/B = A
 		B.HasProximity(AM)
 
+	if(explosion_level && ismob(AM))
+		AM.ex_act(explosion_level)
```

Comment:
so when you enter a turf, you go boom, then when you enter it again, before the explosion wakes up, you go boom again, rinse repeat for who knows how long when explosions take a fair amount of time to process.

A mob shouldn't be getting hit by the same explosion twice, your change would make that happen. You are gonna need to find some way to track the mob.

Also, this wouldn't work if two explosions happen at the same time, such as exploding 5 tank transfer valves via signal that are all in the same backpack.

And this wouldn't work with one explosion because fuel tanks. the fuel tank would explode, overriding the vars you set on the turf.